task_geo/data_sources/nasa/__main__.py

```diff
@@ -0,0 +1,34 @@
+"""Example utilization of the NASA data sourcing."""
+
+from nasa import nasa_meteo_data
```

Comment:
Let's do this move in a second PR, my fork is way behind the master rn, it would just be a mess. I need to refork after this PR is through.

Reply:
Don't refork! use multiple [remotes](https://git-scm.com/docs/git-remote).
You can update your fork easily with:
```
git remote add main git@github.com:CoronaWhy/task-geo.git # This will add the remote named `main`pointing to the original repo, not your fork
git checkout master # Moving to master branch
git pull main # Pulling the changes from the main repository (There should be no problems here)
git checkout $my_branch # Going back to you branch
git merge master # Merge the changes back into your branch
```